### PR TITLE
🎨 Use relative request instead of absolute

### LINKED ISSUE #28: Not working when using a remote server
I want to use this app deployed in a raspberry pi as my headless remote server and controlled using my main pc. 

When I connect remotely using `http://RASPBERRY_IP:5555` the application won't work because all the API request are hard coded to request  `http://localhost:5555`.

I think that one can use relative request in the frontend and let the backend to establish the hostname using the variable SERVER_HOST.

### PR DESCRIPTION
**Pull request by cristianpb**
There is no need to hardcode the hostname in the frontend app, one can use relative requests.

Fix #28

### CODE REVIEW

File: webui/src/components/Record/Record.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ const Record = (props) => {
     }
     console.log(filename);
     axios
-      .post(`${process.env.REACT_APP_API_URL}` + url, {
+      .post(`/` + url, {
```
Comment (mkaliberda) [inline webui/src/components/Record/Record.js:60]: ``` `/${url}` ```

File: webui/src/components/Record/Record.js [hunk 2]
```diff
@@ -80,8 +80,8 @@ const Record = (props) => {
   };
 
   const handleDownloadRequest = (event, filename) => {
-    console.log(`${process.env.REACT_APP_API_URL}download/` + filename);
-    fetch(`${process.env.REACT_APP_API_URL}download/` + filename, {
+    console.log(`/download/` + filename);
+    fetch(`/download/` + filename, {
```
Comment (mkaliberda) [inline webui/src/components/Record/Record.js:84]: ``` `/download/${filename}` ```

**Review by mkaliberda — APPROVED** [review]

**Comment by cdknorow** [comment]
Thanks @cristianpb!  See @mkaliberda comments and remove compiled .js files (we'll do that on our side) and we will merge.

**Comment by cristianpb** [comment]
Done.
Thanks for the review

**Comment by mkaliberda** [comment]
Thank you, Cristian @cristianpb !

**Review by mkaliberda — APPROVED** [review]

**Pull request merged**